generator/config/query/and.yaml

```diff
@@ -8,7 +8,8 @@ description: |
     Joins query clauses with a logical AND returns all documents that match the conditions of both clauses.
 arguments:
     -
-        name: expression
+        name: queries
         type:
             - query
         variadic: array
+        variadicMin: 2
```

Comment:
Why `variadicMin: 2`? While `$and` rejects an empty list on the server side, it works just fine with a single query. Either way, I'm not sure we need to check for this at all. Note this applies to `$or` and `$nor` as well:
```
Enterprise test> db.test.find({$and: []})
MongoServerError: $and/$or/$nor must be a nonempty array
Enterprise test> db.test.find({$and: [{foo: { $exists: false }}]})
// No error
```

Reply:
I have this feature for checking that is used in other places. I'll reduce to 1.